app/assets/javascripts/discourse/app/mixins/composer-upload-uppy.js

```diff
@@ -546,7 +545,7 @@ export default Mixin.create(ExtendableUploader, UppyS3Multipart, {
   },
 
   _reset() {
-    this._uppyInstance?.reset();
+    this._uppyInstance?.cancelAll();
```

Comment:
Have they removed `.reset()` in the new version? We may need to change this in plugins too if that is the case. IIRC `reset()` did more "stuff" than `cancelAll()`

Reply:
According to the migration docs (https://uppy.io/docs/migration-guides.html#Remove-reset-method)

> Remove `reset()` method.
>
> It’s a duplicate of `cancelAll`, but with a less intention revealing name.
> To migrate: use `cancelAll`.

I'll take a look for instances of `reset` in plugins!

Reply:
Ah good to know, thanks for that :+1: